app/src/main/java/com/atom/android/lebo/data/datasource/forgotpassword/ForgotPasswordDataSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.atom.android.lebo.data.datasource.forgotpassword
+
+import com.atom.android.lebo.base.BaseResponse
+import com.atom.android.lebo.data.api.service.ForgotPasswordService
+import io.reactivex.rxjava3.core.Single
+
+class ForgotPasswordDataSource(private val forgotPasswordService: ForgotPasswordService) :
```

Comment:
```suggestion
class ForgotPasswordRemoteDataSource(private val forgotPasswordService: ForgotPasswordService) :
```
mình có 2 source data là remote và local mà e